packages/dd-trace/src/proxy.js

```diff
@@ -155,6 +156,18 @@ class Tracer extends NoopProxy {
         if (config.dynamicInstrumentation.enabled) {
           DynamicInstrumentation.start(config, rc)
         }
+
+        if (config.flaggingProvider.enabled) {
+          rc.setProductHandler('FFE_FLAGS', (action, conf) => {
+            // Feed UFC config directly to flagging provider
+            if (action === 'apply' || action === 'modify') {
+              this.flaggingProvider._setConfiguration(conf.flag_configuration)
+            } else if (action === 'unapply') {
+              // For now, just pass empty config
+              this.flaggingProvider._setConfiguration({})
+            }
+          })
```

Comment:
FYI, the remote config manager doesn't do any merging of received configurations, which means if the backend sends multiple "FFE_FLAGS" (ask whoever is implementing this in the backend to know if it's possible), then each FFE_FLAGS payload will override the previous one, without consolidating them all into one.

Also, line 167 means you'll completely remove the configuration right ? if you have a default/initial config set, you might wanna restore that ?

Just my 2 cents from a RC connoisseur :)

Reply:
@leoromanovsky to confirm: Does the FFE backend send multiple separate FFE_FLAGS configs, or does it always send one consolidated config with all flags? If it sends multiple to represent one full payload, then we will need to implement merging logic

Reply:
A single configuration will have all the flags for an environment; no need to merge.

Reply:
"Also, line 167 means you'll completely remove the configuration right ? if you have a default/initial config set, you might wanna restore that ?"

I don't think there's any current plans to use the 'unapply' action. I think it makes sense to probably remove the handling of it and add it as needed